xwiki-pro-macros-confluence-bridges/xwiki-pro-macros-confluence-bridges-ui/src/main/resources/Confluence/Macros/ConfluenceDetailsSummary.xml

```diff
@@ -309,7 +309,7 @@ Bridges are there for compatibility with content migrated from Confluence and t
         #if ($foreach.first)
           |=$title##
           #foreach ($cell in $row)
-            |=$services.rendering.escape($cell, $xwiki.currentContentSyntaxId)##
+            |=$cell##
```

Comment:
One interesting think to check out is the following:

* UserA with no programming rights creates PageA with some malicious code inside it (the code should not execute because he has no programming rights)
* UserA created PageB and inserts detailssummary macro inside that will display PageA.
* Admin visits PageB

Does the code execute? Im not sure how the programming rights propagate. If the Admin modifies PageB, does the code execute?

Reply:
Hi! The code is executed only if the admin goes to the page with the malicious code and resaves it.
https://github.com/user-attachments/assets/696a8279-6537-489b-8655-2f6fef4c3913

Reply:
So the Admin resaved PageB (with details summary)? Or resaved PageA? (with the malicious code)

It was hard for me to see due to cropping

Reply:
PageA